### DISCUSSION TITLE
Cmax Tmax poor fitting

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1247] [Questions & Problems]
When extracting blood drug concentration data, it is found that there are differences in Cmax and Tmax of the same dose in different literatures, but the parameters used in simulation are the same, which will lead to poor fitting. How to solve this problem？
![Cmax1](https://user-images.githubusercontent.com/95362728/178167970-08d9e83e-25af-42eb-82a1-290579660160.jpg)
![Cmax2](https://user-images.githubusercontent.com/95362728/178167976-568673c1-e87e-4d7a-b0a5-6d4ed170698e.jpg)
![Cmax3](https://user-images.githubusercontent.com/95362728/178168063-1f77ec65-10d4-4f05-bab3-1c3aaf517aa3.jpg)

**Comment by prvmalik** [comment]
Different patient population?

Small number of subjects and random variability?

Different drug assay?

Experimental error?

Different formulation?

By the way, the last screenshot you attached should not be comparable to the first - this appears to be a multiple dose study while the first may be a single dose study.

**Reply by WTY-tt** [reply]
Thank you very much for your answer. I also consider these reasons, but there seems to be no good solution.
